### ISSUE TITLE
modelcheck: plugins property processed incorrect

### ISSUE THREAD

**Issue by arimer** [mbeddr/mps-gradle-plugin#76]
The  modelcheck plugin assumes that all custom plugins configured for the `plugins` property
are placed under a MPS/plugins location which is not required to just load the plugins. 
Instead it should be possible to use the absolute path to determine the location of the plugins.

```
modelcheck {
    plugins = [new Plugin("my.plugin", "my.plugin")]
    ...
}
```
If the custom plugins are not placed under the relative MPS location, 

These changes seem to be wrong: https://github.com/mbeddr/mps-gradle-plugin/commit/16bef1eb2d805b582701df6282739acdc91bd975#r42228923

Loading of custom plugins was supported in the following version of this plugin: 1.2.177.46341c0

**Comment by arimer** [comment]
I've updated the description and the name of the issue. It was not about the `pluginsLocation` property, but about the `plugins` property. 
A possible workaround would be to place the required plugin (which is configured in the `plugins` property) under the expected **mps/plugins** location during the build.

**Comment by arimer** [comment]
fixed by #80
